### ISSUE TITLE
PMPS override daily digest

### ISSUE THREAD

**Issue by ZLLentz** [pcdshub/lcls-twincat-pmps#96]
There should be some mechanism to alert everyone daily that there are pmps overrides active, as per the morning tailgate today

**Comment by ghalym** [comment]
Would a logged message be a good mechanism.
The logger emits these message every 7 AM maybe?

**Comment by ZLLentz** [comment]
Yes, I think that's something we can tie to a notification in grafana.
